<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UcusProgrami</class>
 <widget class="QMainWindow" name="UcusProgrami">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_ucus_id">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sorgula">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sonuc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_grafik" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <zorder>lbl_sonuc</zorder>
   <zorder>widget_grafik</zorder>
   <zorder>txt_ucus_id</zorder>
   <zorder>btn_sorgula</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
